JarrydWorland/2021_Capstone_GP2_Team
# Refactor weapon polish

## Changes include
- Added projectile sprites (c258bf6)
- Added RapidFireWeapon (8c801f1)
- Tweaked projectiles (c495b90)

## How to test
Place a WeaponRapidFire, ensure the default weapon and the WeaponRapidFire work as intended.
